app/Jobs/ApplicationDeploymentJob.php

```
@@ -2447,14 +2631,108 @@ private function generate_build_env_variables()
+                    } else {
+                        $dockerfile->splice(1, 0, ["ARG {$env->key}={$env->real_value}"]);
+                    }
+                }
+            }
+            $dockerfile_base64 = base64_encode($dockerfile->implode("\n"));
+            $this->execute_remote_command([
+                executeInDocker($this->deployment_uuid, "echo '{$dockerfile_base64}' | base64 -d | tee {$this->workdir}{$this->dockerfile_location} > /dev/null"),
+                'hidden' => true,
+            ]);
+        }
+    }
```

PHPMD (2.15.0)

The method add_build_env_variables_to_dockerfile is not named in camelCase. (undefined)

(CamelCaseMethodName)